tests/tests_integration/test_api.py

```diff
@@ -1,11 +1,13 @@
+from . import missing_app_tokens
 from mkmsdk.api import Api
-from . import IntegrationTest
 
 
-class ApiTest(IntegrationTest):
-    def setUp(self):
```

Comment:
Here we can discuss a lot. What I'm saying applies to all tests and is just *an opinion* about how to write readable tests.

First, I usually avoid to create classes (in this case ``TestApi``) when using ``pytest`` because you really don't need them, unless you're grouping together different set of tests in the same file.

**And what about the ``setup_method``?**
I usually avoid it because it doesn't make clear what the test is doing. If you have a very long file and you're fixing a broken test, you have to read the test, go at the top of the file looking for what the ``setUp`` method does and then go back to the test trying to get what it *really* does.

So, if you have 4 tests that must use the ``Api(sandbox_mode=True)`` just declare it in each test. This prevents any side effect making the test more readable, because everything is needed in the test is in the test itself.

**And what about DRY?**
Simply use ``pytest`` [fixtures][1] and "inject" them in your tests.

To sum up, I suggest to:
* remove the class "wrapper" where it applies
* remove all ``setup_method``
* put what the ``setup_method`` does in each test
* if you think that it could be a reusable component, create a fixture

[1]: http://doc.pytest.org/en/latest/fixture.html